main/zoe/guide/contracts/simple-exchange.md

```diff
@@ -127,8 +127,8 @@ const bobPricePayout = await bobPricePayoutP;
 const bobSimGainAmount = await E(bobSimPurse).deposit(bobPricePayout);
 ```
 
-And Alice gets her payouts the same way. (The choice of getPayouts() vs
-getPayout(keyword) is purely for convenience).
+Alice gets her payouts the same way. (The choice of `getPayouts()` vs
+`getPayout(keyword)` is purely for convenience).
```

Comment:
This didn't sound smooth when I wrote it, and now that I'm seeing it again, I want to improve it.
```suggestion
Alice gets her payouts the same way. (The choice between `getPayouts()` and
`getPayout(keyword)` is based on which is more convenient in each circumstance).
```